src/form/en_ivr.jl

```diff
@@ -0,0 +1,1763 @@
+    var(pm, nw, :cid_bus)[id] = _merge_bus_flows(pm, [cid..., -sum(cid)], connections)
+end
+
+
+"""
+	function constraint_mc_load_current_delta(
+		pm::AbstractQuadraticExplicitNeutralIVRModel,
+		nw::Int,
+		id::Int,
+		connections::Vector{Int};
+		report::Bool=true,
+		bounded::Bool=true
+	)
+
+For quadratic IVR models with explicit neutrals,
+create expressions for the terminal current flows `:crd_bus` and `:cid_bus`
+for delta-connected loads
+"""
+function constraint_mc_load_current_delta(pm::AbstractQuadraticExplicitNeutralIVRModel, nw::Int, id::Int, connections::Vector{Int}; report::Bool=true, bounded::Bool=true)
+    crd = var(pm, nw, :crd, id)
+    cid = var(pm, nw, :cid, id)
+    Md = _get_delta_transformation_matrix(length(connections))
+    var(pm, nw, :crd_bus)[id] = _merge_bus_flows(pm, Md'*crd, connections)
+    var(pm, nw, :cid_bus)[id] = _merge_bus_flows(pm, Md'*cid, connections)
+end
+
+
+# TRANSFORMER
+
+# TRANSFORMER - Variables
+
+"""
+	function variable_mc_transformer_current(
+		pm::AbstractExplicitNeutralIVRModel;
+		nw::Int=nw_id_default,
+		bounded::Bool=true,
+		report::Bool=true,
+		kwargs...
+	)
+
+For IVR models with explicit neutrals,
+create transformer current variables `:crt` and `:cit`,
+and placeholder dictionaries for the terminal current flows `:crt_bus` and `:cit_bus`
+"""
+function variable_mc_transformer_current(pm::AbstractExplicitNeutralIVRModel; nw::Int=nw_id_default, bounded::Bool=true, report::Bool=true, kwargs...)
+    variable_mc_transformer_current_real(pm, nw=nw, bounded=bounded, report=report; kwargs...)
+    variable_mc_transformer_current_imaginary(pm, nw=nw, bounded=bounded, report=report; kwargs...)
+
+    var(pm)[:crt_bus] = Dict{Tuple{Int,Int,Int}, Any}()
+    var(pm)[:cit_bus] = Dict{Tuple{Int,Int,Int}, Any}()
+end
+
+
+# TRANSFORMER -  Variable - Non-linear
+
+"""
+	function variable_mc_transformer_power(
+		pm::AbstractNLExplicitNeutralIVRModel;
+		nw::Int=nw_id_default,
+		bounded::Bool=true,
+		report::Bool=true,
+		kwargs...
+	)
+
+For non-linear IVR models with explicit neutrals,
+no power variables are required.
+"""
+function variable_mc_transformer_power(pm::AbstractNLExplicitNeutralIVRModel; nw::Int=nw_id_default, bounded::Bool=true, report::Bool=true, kwargs...)
+    # do nothing    
+end
+
+
+# TRANSFORMER - Variable - Quadratic
+
+"""
+	function variable_mc_transformer_power(
+		pm::AbstractQuadraticExplicitNeutralIVRModel;
+		nw::Int=nw_id_default,
+		bounded::Bool=true,
+		report::Bool=true,
+		kwargs...
+	)
+
+For quadratic IVR models with explicit neutrals,
+creates transformer power variables `:pt` and `:qt`
+"""
+function variable_mc_transformer_power(pm::AbstractQuadraticExplicitNeutralIVRModel; nw::Int=nw_id_default, bounded::Bool=true, report::Bool=true, kwargs...)
+    variable_mc_transformer_power_active(pm, nw=nw, bounded=bounded, report=report; kwargs...)
+    variable_mc_transformer_power_reactive(pm, nw=nw, bounded=bounded, report=report; kwargs...)
```

Comment:
should be `real` and `imaginary` instead of `active` and `reactive`, to be consistent with other naming conventions